package.json

```diff
@@ -24,7 +24,9 @@
     "deploy:staging": "firebase deploy --only hosting:staging-target",
     "eject": "react-scripts eject",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx ./src --ignore-path .gitignore",
-    "precommit": "lint-staged"
+    "postinstall": "npm run setup:maplibre",
```

Comment:
the default manager for this project is "yarn" so let's use that here as well :)

Reply:
Good catch!

Reply:
fixed